the-exterminators/error-annihilator
# Sp-8 Projekt Grundgerüst erstellen und auf GitHub hochladen

## Linked Issues

## PR Description

## Changes


## How to test

1. fetch Branch
2. Open Project in IDE (I recommend IntelliJ)
3. Check if Project Builds and if vaadin classes are imported correctly

### Expected behaviour:
Vaadin Classes should be imported without any failures

## Checklist author


## Checklist reviewer/test-engineer



## Closes Issues
resolves SP-8
